docs/automations/index.md

```diff
@@ -0,0 +1,23 @@
+---
+icon: kolena/rocket-16
+hide:
+  - toc
+---
+
+# :kolena-rocket-20: Advanced Usage
```

Comment:
This should use a different icon and page title from the other "advanced usage" page. However I'd recommend removing this "automations" section altogether, and just grouping everything under "advanced usage".